src/session.ts

```diff
@@ -664,7 +739,7 @@ function onLocalHold(this: WebPhoneSession): boolean {
 
 function addTrack(this: WebPhoneSession, remoteAudioEle, localAudioEle): void {
     const pc = this.sessionDescriptionHandler.peerConnection;
-
+    const mediaCheckTimer = 2000;
```

Comment:
Constants should be defined outside functions, at the top of the file. Otherwise you initialize it every time without a need.

Reply:
Oh yeah correct! will move it out.